00-RELEASENOTES

```diff
@@ -9,6 +9,46 @@ CRITICAL: There is a critical bug affecting MOST USERS. Upgrade ASAP.
 SECURITY: There are security fixes in the release.
 --------------------------------------------------------------------------------
 
+================================================================================
+Valkey 8.0.2  -  Released Mon 6 Jan 2025
```

Comment:
nit: based on previous release notes.

```suggestion
Valkey 8.0.2  -  Released Mon 06 Jan 2025
```

Reply:
The other 8.0 releases don't use two digits, so I'm not sure there is a pattern.

Reply:
Ah, I was doing it for 7.2 and saw the pattern. Ignore.